attestation/factory.go

```diff
@@ -41,6 +41,8 @@ type Subjecter interface {
 	Subjects() map[string]cryptoutil.DigestSet
 }
 
+// NOTE: not sure on the name of this interface, however I can't think of an alternative for now
```

Comment:
The weird world of Go interface naming https://go.dev/doc/effective_go#interface-names